conversion.ipynb

```diff
@@ -234,7 +234,6 @@
         "    predict_fn,\n",
         "    with_gradient=with_gradient,\n",
         "    polymorphic_shapes=[None, \"b, 224, 224, 3\"],\n",
-        "    enable_xla=True,\n",
```

Comment:
I'd prefer to leave this be the default value (True). It is possible that the `enable_xla` parameter will change in the future, so the code is more future-proof without it.